internal/server/proto/server.proto

```diff
@@ -195,6 +195,9 @@ service Waypoint {
   // GetRunner gets information about a single runner.
   rpc GetRunner(GetRunnerRequest) returns (Runner);
 
+  // ListRunners gets information about existing runners.
```

Comment:
Nitpick but I would change these docs to note that this only lists runners that are currently running and registered, i.e. explicitly omits on-demand runners that are not currently running.